### ISSUE TITLE
More descriptive UBI parameter names

### ISSUE THREAD

**Issue by MaxGhenis** [open-source-economics/Tax-Calculator#1807]
Minor enhancement consideration here: how would you all feel about more descriptive UBI parameter names? They're currently:

```
_UBI1  # Under 18, aligns with nu18.
_UBI2  # 18-20, aligns with n1821.
_UBI3  # Over 20, aligns with n21.
```

`n1821` will be renamed to `n1820` per https://github.com/open-source-economics/Tax-Calculator/issues/1669. 

It would be nice to have the UBI params indicate their association to the three variables, for example as `_UBI_u18`, `_UBI_1820`, and `_UBI_21`. Or a bit more verbosity could be clearer still IMO, e.g. `_UBI_0_17`, `_UBI_18_20`, and `_UBI_21_plus`, though it might make sense to name the `n*` variables accordingly too in that case.

**Comment by martinholmer** [comment]
@MaxGhenis said:
> It would be nice to have the UBI params indicate their association to the three [UBI-related age-count] variables, for example as `_UBI_u18`, `_UBI_1820`, and `_UBI_21`.

This seems like an improvement to me, but perhaps I'm missing something.

Unless I hear objections in the next few days, I'll begin next week working on a pull request to make the above name changes.  

This approach has the virtue of not requiring any input variable name changes other than the logically-separate taxdata fix to the bug described in #1669.

**Comment by martinholmer** [comment]
Merge of pull request #1813 resolves issue #1807.
